cs/src/Connections/TunnelClient.cs

```diff
@@ -240,6 +243,12 @@ protected async Task StartSshSessionAsync(Stream stream, CancellationToken cance
     private void OnSshSessionDisconnected(object? sender, EventArgs e) =>
         StartReconnectTaskIfNotDisposed();
 
+    private void OnSshSessionClosed(object? sender, EventArgs e)
```

Comment:
How is it different from `SshSession_Closed` ?

Reply:
I will combine them.

Reply:
Done.